dev/reference/configuration/tidb-server/configuration.md

```diff
@@ -125,9 +121,14 @@ TiDB is configurable using command-line flags and environment variables. The def
 
 - The status report port for TiDB server
 - Default: "10080"
-- This is used to get server internal data. The data includes [Prometheus metrics](https://prometheus.io/) and [pprof](https://golang.org/pkg/net/http/pprof/).
-- Prometheus metrics can be got through `"http://host:status_port/metrics"`.
-- Pprof data can be got through `"http://host:status_port/debug/pprof"`.
+- This port is used to get server internal data. The data includes [Prometheus metrics](https://prometheus.io/) and [pprof](https://golang.org/pkg/net/http/pprof/).
+- Prometheus metrics can be accessed by `"http://host:status_port/metrics"`.
```

Comment:
```suggestion
- Prometheus metrics can be accessed by <http://host:status_port/metrics>.
```

Reply:
@yikeke It's written as **"http://host:status_port/metrics"** in the config file. So let's keep the quotation marks.

Reply:
Okay.